client/adventureMap/TurnTimerWidget.h

```diff
@@ -39,13 +39,13 @@ class TurnTimerWidget : public InterfaceObjectConfigurable
 	std::set<int> notifications;
 	
 	std::shared_ptr<DrawRect> buildDrawRect(const JsonNode & config) const;
-	
+
 public:
 
 	void show(Canvas & to) override;
 	void tick(uint32_t msPassed) override;
 	
-	void setTime(int time);
+	void setTime(PlayerColor player, int time);
```

Comment:
could you please take signature from https://github.com/vcmi/vcmi/pull/2674 so we can avoid merge conflicts?

Reply:
I can, but shouldn't this be more logical order?

Reply:
In this case setPlayerTime may be a better name
Btw will we have a separate timer for each player now as later they may have to act in parallel?

Reply:
If we want to do it "hd mod way" then yes.
Although HD mod displays timers for *all* players, including those who are not making turn - it always shows base timer I think.

Reply:
I do not know HD mod well, but came into same conclusion)